src/jobs/utils/getListings.ts

```diff
@@ -94,6 +94,8 @@ async function getValidityStatuses(department?: Office): Promise<Offer[]> {
       throw new Error('Could not fetch data from Recruitee');
     }
 
+    offer.careers_apply_url = insertSwedishLangUrl(offer.careers_apply_url);
+
```

Comment:
Could do this to make Mikael happy
```suggestion

    const newOffer = {...offer, careers_apply_url: insertSwedishLangUrl(offer.careers_apply_url)};

```